crates/miden-testing/src/tx_context/context.rs

```diff
@@ -131,13 +135,10 @@ impl TransactionContext {
         let source_manager = Arc::clone(&self.source_manager);
         let tx_executor = TransactionExecutor::new(&self, authenticator).with_debug_mode();
 
-        maybe_await!(tx_executor.execute_transaction(
-            account_id,
-            block_num,
-            notes,
-            tx_args,
-            source_manager
-        ))
+        // TODO: Make the function async, but this is easier for the POC stage.
+        tokio::runtime::Builder::new_multi_thread().build().unwrap().block_on(
+            tx_executor.execute_transaction(account_id, block_num, notes, tx_args, source_manager),
+        )
```

Comment:
I'm taking a lot of shortcuts to get something to work, and this is one of them. This happens in `TransactionContext::execute`. This uses a multi-threaded runtime just to check that it works and can be changed to single threaded if necessary.